app/javascript/client/error/ErrorPage.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+
+const ErrorMessage = styled.div`
+    width: 250px;
+    height: auto;
+    margin: 0 auto 24px auto;
+`;
+
+interface ErrorPageProps {}
+
+export const ErrorPage: FC<ErrorPageProps> = () => {
+
+    return (
+        <>
+            <ErrorMessage>
+                Oops! Something went wrong on our end. Our apologies.
```

Comment:
We can totally add the error monkey here
![image](https://user-images.githubusercontent.com/55555255/89430441-c00edb80-d70c-11ea-90fd-6ca7a703cb22.png)

Reply:
:joy: I hadn't seen that guy before ... I like him!! Agreed, will add in a future PR 😆